assets/js/main.js

```diff
@@ -353,4 +361,13 @@ $(function () { // This prevents global vars
       });
     }
   });
+});
+
+/* This adds hover anchor links to all headers */
+$(function() {
+  anchors.options = {
+    placement: 'left'
+  };
+  anchors.add();
+  anchors.add('.terms .terms-row strong');
```

Comment:
Can we be a little more targeted about applying this style? `terms` is a generic enough word that I could imagine it being used in another context and getting a surprise anchor applied to it. Perhaps you can give each page's `<body>` an `id` and then specifically declare the `id`s as part of the CSS selector here.